test/testdata/wrapper/lts-11.14/stack.yaml

```diff
@@ -1 +0,0 @@
-resolver: lts-11.14
```

Comment:
you need to add a stack.yaml files, otherwise the different ghc versions cant be detected. This is very likely the reason the test fails

Reply:
A stack.yaml file for each wrapper test.

Reply:
This was moved to `test/testdata/wrapper/lts-14.18/stack.yaml`, so testing only needs 8.6.5 and 8.8.1.
Hie is already a heavy project, cutting a few gig counts on 256G laptops.

Reply:
In tests, you are also using: ` withCurrentDirectory "test/testdata/wrapper/8.8.1"` in which was no stack.yaml

Reply:
Oops, I missed that those were not included due to being in `.gitignore`. Thanks